setup.py

```diff
@@ -8,5 +21,9 @@
 	author='Bosak Adam',
 	author_email='bosadam@seznam.cz',
 	url='https://github.com/bosakad/GP-Optimization',
-	packages=find_packages()
+	packages=find_packages(),
+	cmdclass={
+        'clean': CleanCommand,
+    }
+
```

Comment:
I would suggest exclude those directories, where you have no packages. You packages are in `/src`. So you don't want that whatever is in `/output` and `/examples` will be written to a PATH. Particularly, this `/output` directory may have lots of different data that will probably appear there from run to run.

To do this, you should write:
```python
packages=setuptools.find_namespace_packages(exclude=["tests", "docs","examples","inputs_outputs"]),
```